doc/nrf/releases/release-notes-latest.rst

```diff
@@ -441,6 +444,11 @@ The following list summarizes the most important changes inherited from upstream
     * Added experimental support for Extended Scanning with duration and period parameters (without active scanning for scan response or chained PDU).
     * Added experimental support for Periodic Advertising and Periodic Advertising Synchronization Establishment.
 
+  * Bluetooth HCI:
+
+    * Added support for ISO packets to the RPMsg driver.
+    * Added possibility of discarding advertising reports to avoid timeouts in the User Channel, STM32 IPM, and SPI based drivers.
```

Comment:
```suggestion
    * Added the possibility of discarding advertising reports to avoid time-outs in the User Channel, STM32 IPM, and SPI based drivers.
```

Reply:
Applied.